pkg/cmd/admin/registry/registry.go

```diff
@@ -426,6 +428,18 @@ func (opts *RegistryOptions) RunCmdRegistry() error {
 				},
 			},
 		})
+	} else if opts.Config.Deployment {
```

Comment:
you're right, but the point of this PR was mostly to ignite discussion about whether we want to move to deployments as default for router and registry...

synce both of them don't use any of the deployment config features (hooks/triggers) i think having them as DC by default is overkill (it requires deployer pod, might fail to deploy, etc...)